napari/components/dims.py

```diff
@@ -65,6 +67,8 @@ class Dims(EventedModel):
     displayed_order : tuple of int
         Order of only displayed dimensions. These are calculated from the
         ``displayed`` dimensions.
+    thickness : tuple of float
+        Thickness of the slice in each dimension, measured in number of steps.
```

Comment:
Should this be in world coordinates and a tuple of float? Or in slider coordinates (steps) and a tuple of int? World coordinates might be better here. What does @grlee77 think?

Reply:
At first I had world coordinates, but it felt more intuitive to think in steps when interacting with it from the gui.

Reply:
yeah, i think you need steps if there is a slider behind it, but we might need to then think about some slider to world mapping (maybe same as main sliders). @jni can give some more history on how we used to think about this with the range slider. curious what he recommends here

Reply:
@sofroniewn, I can take a look by Monday

Reply:
Since we have only a single `Dims` object on the `ViewerModel` (`viewer.dims`), we do have consistent steps across all layers, so I think it is okay. As stated, it does seem like steps, rather than world coordinates fits in easier with the sliders added to the `QtDims` object.